CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+### Removed
+
+- Previously deprecated `/gov/jwt_keys/all` has been removed. The `/gov/service/jwk` endpoint to be used instead.
```

Comment:
Is that true for both legacy and new versioned APIs? Should we rev up the API version?

Reply:
Now as you've asked I'm interested if we should have done it when adding issuer to the response, as well as shall we do it here, even if we keep the deprecated endpoint.

Asking because the response format has changed, and I'm not sure about our api versioning policy.